providers/provider_default.go

```diff
@@ -119,6 +119,11 @@ func (p *ProviderData) ValidateGroup(email string) bool {
 	return true
 }
 
+// ValidateUser validates that the user has some required characteristic
+func (p *ProviderData) ValidateUser(user string) bool {
+	return false
```

Comment:
Is this the correct default? `ValidateGroup` has `true` as the default, could you quickly explain your reasoning for me here?

Reply:
Yeah that is weird. Look here: https://github.com/pusher/oauth2_proxy/blob/master/oauthproxy.go#L778
I am allowing `ValidateUser` to side-step `p.Validator(session.Email)` since the user doesn't necessarily have an allowed email (since they are an outside collaborator). Maybe there's a better way to do this